.github/workflows/ci_workflows.yml

```diff
@@ -2,10 +2,6 @@ name: CI
 
 on:
   push:
-    branches:
```

Comment:
Need to remove this before we merge!

Reply:
Done. It's a trivial rebase because I had it as a distinct commit. Actually I thought I had already done so.